### PR TITLE
Feature/legend-layer-citations

### PR DESCRIPTION
**Pull request by Bluesmile82**
Add Legend layer citations

This PR adds to the layers of the legend the source reference and the source link.

## Testing instructions

Activate some layer on the map. You will see the Source if available

## Tracking

https://vizzuality.atlassian.net/browse/RA2-153

### CODE REVIEW

File: frontend/src/views/components/Legend/Legend.component.jsx [hunk 1]
```diff
@@ -98,6 +109,23 @@ const Legend = ({ activeLayers, reorder, loading, toggleLayer, setOpacity }) =>
...
+                                <div className="source">
+                                  <span className="source-bold">
+                                    <T _str="Source:" />{' '}
+                                  </span>
+                                  <span>{sourceReference}</span>
+                                </div>
+                                <a
+                                  className="source-link"
+                                  target="_blank"
+                                  rel="noopener noreferrer"
```
Comment (clementprdhomme) [inline frontend/src/views/components/Legend/Legend.component.jsx:123]: You forgot the `href` attribute.

Comment (Bluesmile82) [reply frontend/src/views/components/Legend/Legend.component.jsx:123]: Thanks! I just updated it

File: frontend/src/views/components/Legend/Legend.component.jsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const Legend = ({ activeLayers, reorder, loading, toggleLayer, setOpacity }) =>
   const [opened, toggleOpen] = useToggle(true);
   const onDragEnd = useCallback(
     ({ source, destination }) => destination && reorder(source.index, destination.index),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [],
   );
```
Comment (davidsingal) [inline frontend/src/views/components/Legend/Legend.component.jsx:18]:
```suggestion
  const onDragEnd = useCallback(
    ({ source, destination }) => !!destination && reorder(source.index, destination.index),
    [reorder],
  );
```
why not include `reorder` in the list of dependencies of the hook?

**Comment by Bluesmile82** [comment]
Ready again!

**Pull request merged**